### PR TITLE
read words

### PR DESCRIPTION
**Pull request by kiennt175**
## Related Tickets:
- [#28917](https://edu-redmine.sun-asterisk.vn/issues/28917)

## Notes Impacted Areas:
- N/A.

## Attachments:
- N/A.

### CODE REVIEW

**Comment by kiennt175** [comment]
ready

File: app/Http/Controllers/WordController.php [hunk 1]
```diff
@@ -58,4 +58,73 @@ public function store(Request $request)
...
+            ['type_id', '=', $typeId],
+        ])->get()->isNotEmpty();
+        if ($checkWord) 
+            return back()->with('message', trans('words.check_word'));
+        try {
+            $word = Word::find($wordId);
+            $types = $word->types()->get()->toArray();
+            $totalRecords = count($types);
+            $word->types()->wherePivot('type_id', $typeId)->detach();
+            if ($totalRecords === 1) {
```
Comment (huyenttm-0901) [inline app/Http/Controllers/WordController.php]: đưa số vào config

File: resources/views/layouts/layout.blade.php [hunk 2]
```diff
@@ -68,11 +68,11 @@
                                     </a>
                                     <div class="dropdown-menu" aria-labelledby="navbarDropdown">
                                         <a class="dropdown-item" 
-                                        href="{{ route('user.change-language', ['vi']) }}">
+                                        href="{!! route('user.change-language', ['vi']) !!}">
                                             {{ trans('home.vi') }}
                                         </a>
                                         <a class="dropdown-item" 
-                                        href="{{ route('user.change-language', ['en']) }}">
+                                        href="{!! route('user.change-language', ['en']) !!}">
```
Comment (huyenttm-0901) [inline resources/views/layouts/layout.blade.php]: sao lại dùng {!!

File: resources/views/words.blade.php [hunk 3]
```diff
@@ -0,0 +1,164 @@
...
+    </div>
+    <div class="container balance">
+        <div class="card">
+            <div class="card-header">{{ trans('words.import_label') }}</div>
+            <div class="card-body ">
+                <div class="col">
+                    <form action="" method="post" enctype="multipart/form-data">
+                        @csrf
+                        <input type="file" name="import" 
+                        accept="application/vnd.openxmlformats-officedocument.spreadsheetml.sheet" required>
```
Comment (huyenttm-0901) [inline resources/views/words.blade.php]: indent 4 space

File: resources/views/words.blade.php [hunk 4]
```diff
@@ -0,0 +1,164 @@
...
+                                            </a>
+                                            <form action="{{ route('deleteWord', 
+                                            ['wordId' => $word->id, 'typeId' => $type->pivot->type_id]) }}" 
+                                            method="post">
+                                                @method('DELETE')
+                                                @csrf
+                                                <div class="modal fade" id=
+                                                "exampleModalCenter{{ $word->id }}{{ $type->pivot->type_id }}" 
+                                                tabindex="-1" role="dialog" 
+                                                aria-labelledby="exampleModalCenterTitle" aria-hidden="true">
```
Comment (huyenttm-0901) [inline resources/views/words.blade.php]: indent 4 spaces

**Comment by kiennt175** [comment]
ready

**Pull request merged**